src/js/settings-list.js

```diff
@@ -429,6 +429,12 @@ module.exports = [
         }
     },
     {
+        name: 'Hide Last Broadcast Banner',
```

Comment:
Then have it remember it for the single page view. As long as the user can close it when it opens, there should be no reason why they can't close it instead of complaining it's there.